build/tasks/prepare-release.js

```diff
@@ -60,7 +70,7 @@ gulp.task("commit-changes", () => {
 	const version = getPackageJsonVersion();
 	return gulp.src(".")
 		.pipe(git.add())
-		.pipe(git.commit(`[Prepare Release] Bumped version to ${version}.`));
+		.pipe(git.commit(`release(changelog): bumped version to ${version} & updated changelog.`));
```

Comment:
1. surround `${version} in backticks also (you have to escape them e.g. 

```md
\`${version}\`
```

2. remove period (.)
3. (optional) perhaps change `&` to `and` or `+` 😛

Reply:
done